src/index.ts

```diff
@@ -59,7 +59,8 @@ async function metadataAccessor<T>(
       url: `${BASE_URL}/${type}${property}`,
       headers: Object.assign({}, HEADERS, options.headers),
       retryConfig: {noResponseRetries},
-      params: options.params
+      params: options.params,
+      responseType: 'text'
```

Comment:
Is this safe to do for recursive queries?

```
$ curl -H 'Metadata-Flavor: Google' 'http://metadata.google.internal/computeMetadata/v1/instance?alt=text'
Non-recursive directory listings do not support ?alt.
```

Reply:
This code doesn't add `?alt` to the querystring when calling the metadata service.  All it does is tell gaxios to *not* call `JSON.parse` when it gets the result back.

Reply:
👍